docs/content/preview/explore/change-data-capture/_index.md

```diff
@@ -18,25 +18,31 @@ Change data capture (CDC) is a process to capture changes made to data in the da
 
 ### On this page
 
-* [Prerequisites](#prerequisites)
-* [Process architecture](#process-architecture)
-* [Debezium connector for YugabyteDB](#debezium-connector-for-yugabytedb)
-* [Java console client](#cdc-java-console-client)
-* [Tserver configuration](#tserver-configuration)
-* [Consistency semantics](#consistency-semantics)
-* [Performance impact](#performance-impact)
-* [yb-admin commands](#yb-admin-commands)
-* [DDL command support](#ddl-command-support)
-* [Snapshot support](#snapshot-support)
-* [Limitations](#limitations)
+- [What is CDC?](#what-is-cdc)
+  - [Prerequisites](#prerequisites)
+  - [Process Architecture](#process-architecture)
+    - [CDC Streams](#cdc-streams)
+    - [DB Stream](#db-stream)
+  - [Debezium connector for YugabyteDB](#debezium-connector-for-yugabytedb)
+  - [CDC Java console client](#cdc-java-console-client)
+  - [TServer configuration](#tserver-configuration)
+  - [Consistency Semantics](#consistency-semantics)
+    - [Per-Tablet Ordered Delivery Guarantee](#per-tablet-ordered-delivery-guarantee)
+    - [At least Once Delivery](#at-least-once-delivery)
+    - [No Gaps in Change Stream](#no-gaps-in-change-stream)
+  - [Performance Impact](#performance-impact)
+  - [yb-admin commands](#yb-admin-commands)
+  - [DDL command support](#ddl-command-support)
+  - [Snapshot support](#snapshot-support)
+  - [Limitations](#limitations)
 
 ## What is CDC?
 
 The core primitive of CDC is the _stream_. Streams can be enabled/disabled on databases. Every change to a watched database table is emitted as a record in a configurable format to a configurable sink. Streams scale to any YugabyteDB cluster independent of its size and are designed to impact production traffic as little as possible.
```

Comment:
Doesn't really describe CDC. This description looks more applicable to Process Architecture below, where the 2 types of streams are described. I'd move this paragraph to that section, get rid of this heading, and bump the headings up a level.